CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @yterraillon @mikyan0207
```

Comment:
il me semble que ce fichier va dans un répertoire .github à la racine du repo

Reply:
D'après la doc de github, il y a plusieurs possibilités. Je peux le déplacer dans un dossier .github si tu préfères

> To use a CODEOWNERS file, create a new file called CODEOWNERS in the root, docs/, or .github/ directory of the repository

Reply:
ah ok, je ne savais pas. Go for it!